modules/neo4j/src/test/java/org/testcontainers/containers/Neo4jContainerTest.java

```diff
@@ -0,0 +1,68 @@
+package org.testcontainers.containers;
+
+import static org.hamcrest.CoreMatchers.*;
+import static org.junit.Assert.*;
+
+import java.util.Collections;
+
+import org.junit.Test;
+import org.neo4j.driver.v1.AuthToken;
+import org.neo4j.driver.v1.AuthTokens;
+import org.neo4j.driver.v1.Driver;
+import org.neo4j.driver.v1.GraphDatabase;
+import org.neo4j.driver.v1.Session;
+
+/**
+ * Tests of functionality special to the Neo4jContainer.
+ *
+ * @author Michael J. Simons
+ */
+public class Neo4jContainerTest {
+
+    @Test
+    public void shouldDisableAuthentication() {
+
+        try (
+            Neo4jContainer neo4jContainer = new Neo4jContainer().withAdminPassword(null);
+        ) {
+            neo4jContainer.start();
+            try (Driver driver = getDriver(neo4jContainer);
+                Session session = driver.session()
+            ) {
+                long one = session.run("RETURN 1", Collections.emptyMap()).next().get(0).asLong();
+                assertThat(one, is(1L));
+            }
+        }
+    }
+
+    @Test
+    public void shouldRunEnterprise() {
+
+        try (
+            Neo4jContainer neo4jContainer = new Neo4jContainer()
+                .withEnterpriseEdition()
+                .acceptLicense()
```

Comment:
I _assume_ that the Testcontainers project will be permitted to freely use the enterprise edition for local builds and CI testing of this module? 😂

Reply:
I’ll check first thing next week.